### PR TITLE
[release/1.10.0] Update release note

### PR DESCRIPTION
**Pull request by hseok-oh**
Update release note: runtime part

Signed-off-by: Hyeongseok Oh <hseok82.oh@samsung.com>

### CODE REVIEW

File: docs/release/1.10/release-note-1.10.0.md [hunk 1]
```diff
@@ -13,3 +13,13 @@
...
+
+
+## ONE Runtime
+
+### Runtime backend operation support
+  - ACL-CL backend: OneHot
+  - CPU backend: FullyConnected for Float32 16x1 Block Sparsity
+
+### Optimization
+  - Speed up for ReduceSum, StrideSlice in CPU backend
```
Comment (glistening) [inline docs/release/1.10/release-note-1.10.0.md:25]:
```suggestion
  - Speed up for ReduceSum, StrideSlice and BinaryArithmetic in CPU backend
```

**Review by glistening — APPROVED** [review]
LGTM

**Review by chunseoklee — APPROVED** [review]

**Comment by hseok-oh** [comment]
@wateret PTAL

**Review by wateret — APPROVED** [review]
LGTM

**Pull request merged**